### PR TITLE
Circle ci

### PR DESCRIPTION
**Pull request by bumblebee24**

### CODE REVIEW

File: docker-start.sh [hunk 1]
```diff
@@ -3,4 +3,4 @@
 if [ ! -d 'node_modules' ]; then
   npm install
 fi
-npm start
+npm run dev-start
```
Comment (niksajanjic) [inline docker-start.sh]: zasto ovo minjat?

npm start je bolje nego npm run dev-start

File: package.json [hunk 2]
```diff
@@ -4,10 +4,11 @@
   "description": "ECP skeleton for React/Redux web application",
   "scripts": {
     "build": "NODE_ENV=production ./node_modules/webpack/bin/webpack.js --config ./webpack.prod.config.js --progress --profile --colors",
-    "prod": "node prod.js",
```
Comment (niksajanjic) [inline package.json]: Zasto je ovo maknuto?

File: package.json [hunk 3]
```diff
@@ -4,10 +4,11 @@
...
+    "start": "node prod.js",
     "snapshots": "jest -u",
-    "start": "node server.js",
-    "test": "jest"
+    "dev-start": "node server.js",
+    "test": "jest",
+    "postinstall": "npm run build"
```
Comment (niksajanjic) [inline package.json]: What is this line for, can't you just do npm run build directly?

Comment (bumblebee24) [inline package.json]: We need npm start for production so development is changed to npm run dev-start

Comment (niksajanjic) [inline package.json]: That is other question, I asked here about postinstall.

**Comment by niksajanjic** [comment]
LGTM

**Pull request merged**